region/brinstar/blue.json

```diff
@@ -1239,6 +1416,198 @@
+                        "mode": "direct",
+                        "artificialMorph": true,
+                        "mobility": "immobile"
+                      }}
+                    ]}
+                  ],
+                  "note": [
+                    "Overload PLMs with a single Power Bomb by hitting the item at the right end of the room.",
+                    "This can be done with a single precisely placed Power Bomb. There should be 4 empty tiles between Samus and the left of the blocks when the bomb is placed."
+                  ]
+                },
+                {
+                  "name": "G-Mode Morph Power Bomb the Item with Flashing Lights",
+                  "notable": true,
+                  "requires": [
+                    {"itemNotCollectedAtNode": 4},
+                    { "not": "f_ZebesAwake" },
+                    "canRiskPermanentLossOfAccess",
+                    {"comeInWithGMode": {
+                      "fromNodes": [1],
+                      "mode": "direct",
+                      "artificialMorph": true
+                    }},
+                    {"ammo": {"type": "PowerBomb","count": 1}},
+                    {"or": [
+                      "canConsecutiveWalljump",
+                      "SpaceJump"
+                    ]},
+                    {"or": [
+                      "SpringBall",
+                      "Bombs",
+                      {"ammo": {"type": "PowerBomb","count": 1}},
+                      {"enemyDamage": {
+                        "enemy": "Geemer (blue)",
+                        "hits": 1,
+                        "type": "contact"
+                      }},
+                      {"comeInWithGMode": {
+                        "fromNodes": [1],
+                        "mode": "direct",
+                        "artificialMorph": true,
+                        "mobility": "immobile"
+                      }}
+                    ]}
+                  ],
+                  "reusableRoomwideNotable": "Blue Brinstar Energy Tank G-Mode Flashing Lights",
+                  "note": [
+                    "The eye scanners are particularly annoying while in g-mode. They scan Samus with bright flashing lights which remain for a further distance.",
+                    "This is notable so a player can disable having to enter these flashing lights. If disabled, Samus will only require being in g-mode in this room if Zebes is awake.",
+                    "Overload PLMs with a single Power Bomb by hitting the item at the right end of the room.",
+                    "This can be done with a single precisely placed Power Bomb. There should be 4 empty tiles between Samus and the left of the blocks when the bomb is placed."
+                  ]
+                },
+                {
+                  "name": "G-Mode Morph Touch the Item",
+                  "notable": false,
+                  "requires": [
+                    {"itemNotCollectedAtNode": 4},
```

Comment:
This doesn't visit 4 to get the item so I don't think the logic tracks this as a one time only strat.